EIPS/eip-xtndr-family.md

```diff
@@ -0,0 +1,98 @@
+# EIP: XTNDR opcode family
+
+title: XTNDR family opcode
+description: Opcodespace extension via prefix, variadic opcodes and bytecode compression through macros
+author: Brayton Goodall, Mihir Faujdar
+type: Standards Track
+category: Core
+created: 2021-11-30
+
+
+## Table of Contents
+- [Abstract](#Abstract)
+- [Motivation](#Motivation)
+- [Specification](#Specification)
+- [Rationale](#Rationale)
+- [Backwards Compatibility](#Backwards-Compatibility)
+- [Test Cases](#Test-Cases)
+- [Reference Implementation](#Reference-Implementation)
+- [Security Considerations](#Security-Considerations)
+- [Copyright](#Copyright)
+
+## Abstract
+Introduce XTNDR, a prefix opcode to specify the bytes that immediately follow are to be interpreted as a multi-byte opcode. Additionally single-byte opcodes can have their adicity adjusted, in effect allowing in-bytecode macros for more compact functions. A portion of single-byte opcodespace is reserved for 'loading' prespecified multi-byte opcodes such that their usage is not prohibitively expensive. 
+
+
+## Motivation
+
+All EVM opcodes are single-byte, which encourages careful choice when introducing new opcodes. 
+Additionally, high-level blockchain and EVM policies (such as gas prices of operations) often concern tradeoffs between data size, network requirements for propagation and ease of execution. These changes would allow a greater range of space-time tradeoffs of smart contracts, through situationally allowing smaller contract sizes which are more complex to execute. This may allow greater flexibility in high-level policies and eventually network efficiency. An additional factor is to slightly increase the memory burden of the EVM in exchange for reduced storage requirements and disk wear.
+
+This method aims to provide three features in such a way that they may be cohesively applied for maximal elegance, ease of implementation and ultimately efficiency of the protocol as a fundamental infrastructure layer which others build upon. The first and simplest feature is the declaration of prefix bytes to reserve sequences for future multi-byte opcodes.
+
+The second of these three features is the introduction of variadic opcodes through two bytes preceeding a more conventional operation, in order to allow smaller bytecode footprints of highly structured repeated operations.
+
+The third feature is a bytecode compression technique which substitutes multi-byte sequences (of the other two features) for specific single bytes previously declared within a contract, effectively holding a sequence of instructions in it's own memory. 
+
+## Specification
+
+The XTNDR opcode is found at 0xED. 
+
+A sequence beginning with XTNDR is to signify a multi-byte opcode or otherwise an 'extended' operation. For example two-byte sequences starting with ED and followed by a non-numeric value is to be interpreted as a single opcode from 'extended space'.
```

Comment:
To point out a basic mistake: there no such thing as non-numeric byte value. All opcodes / bytes are numbers so you cannot make such distinction. E.g. `XTNDR 3 ADD` is `ED 03 01` so it can also mean `XTNDR SUB 1`.